ingest-doc-blob-document-intelligence-cosmos/default/manifest.json

```diff
@@ -0,0 +1,83 @@
+    "kinds": [
+        "stateful",
+        "stateless"
+    ],
+    "artifacts": [
+        {
+            "type": "workflow",
+            "file": "workflow.json"
+        }
+    ],
+    "images": {
+        "light": "workflow-light",
+        "dark": "workflow-dark"
+    },
+    "parameters": [
+        {
+            "name": "OpenAI_TextEmbedding_Deployment",
+            "displayName": "OpenAI text embedding deployment identifier",
+            "type": "String",
+            "description": "Provide the deployment model for Azure OpenAI.",
+            "required": true
+        },
+        {
+            "name": "Blob_Path",
+            "displayName": "Blob Storage documents path",
+            "type": "String",
+            "description": "Provide the path to the documents in Blob Storage.",
+            "required": true
+        },
+        {
+            "name": "CosmosDB_Database",
+            "displayName": "Cosmos DB name",
+            "type": "String",
+            "description": "Provide the name for the Azure Cosmos DB database.",
+            "required": true
+        },
+        {
+            "name": "CosmosDB_Collection",
+            "displayName": "Cosmos DB collection name",
+            "type": "String",
+            "description": "Provide the name for the Azure Cosmos DB collection.",
+            "required": true
+        },
+        {
+            "name": "CosmosDB_Vector_Path",
+            "displayName": "Vector embeddings path or property",
+            "type": "String",
+            "description": "Provide the path or property for the vector embeddings in the Cosmos DB data items.",
+            "required": true
+        },
+        {
+            "name": "CosmosDB_Text_Path",
+            "displayName": "Property with the document text in data items",
+            "type": "String",
+            "description": "Provide the property that contains the document text in the Cosmos DB data items.",
+            "required": true
+        }
+    ],
+    "connections": {
+        "aoai-connection_#workflowname#": {
+            "connectorId": "/serviceProviders/openai",
+            "kind": "inapp"
+        },
+        "blob-connection_#workflowname#": {
+            "connectorId": "/serviceProviders/AzureBlob",
+            "kind": "inapp"
+        },
+        "cosmosdb-connection_#workflowname#": {
```

Comment:
Can we make sure ALL cosmos db template are using this same key for cosmos DB connection?